locals.tf

```diff
@@ -1,22 +1,17 @@
 locals {
   
   # Useful flags
-  is_private            = var.infra_mode == "private" ? true : false
-  is_standard           = var.infra_mode == "standard" ? true : false
-  create_jump_public_ip = var.create_jump_public_ip == null ? local.is_standard : var.create_jump_public_ip
-  create_nfs_public_ip  = var.create_nfs_public_ip == null ? local.is_standard : var.create_nfs_public_ip
```

Comment:
How do I turn on/off the jump and nfs public IP values? Do I add these variables directly to my terraform vars file? No doc showing how this has changed.

Reply:
The intent is that you can set var.create_jump_public_ip and var.create_nfs_public_ip independently of the cluster endpoint.

Reply:
This code has been released with the `infra_mode` flag already. So somewhere this needs to be addressed. This is going to be a breaking change for folks who may use this feature.  Need to have some docs on this one.

Reply:
since infra_mode was never documented (and never integrated with viya4-deployment), maybe a note in one of our internal Teams channels would suffice here? In short, to get parity with infra_mode="private" you would now set:
cluster_api_mode="private"
create_jump_public_ip=false
create_nfs_public_ip=false (if storage_type="standard")

and possibly
default_public_access_cidrs=[]

Reply:
I am ok with this. Need to set these create_<vm>_public_ip items to `true` for jump server by default in variable file and `false` for nfs server by default. Right now they are both `null` which was used when we were using the local value. All example files should be updated to explicitly set these as well. I believe the CONFIG-VAR.md doc is correct. I think the variables.tf file is not.

Reply:
fixed variable.tf defaults. 
Not sure about the examples. We already set both variables explicitly in most of them. The behavior does not really change here.